hack/generator/pkg/astmodel/definition_name.go

```diff
@@ -29,6 +29,8 @@ func (dn *DefinitionName) Name() string {
 // it is simply a reference to the name.
 var _ Type = (*DefinitionName)(nil)
```

Comment:
We may be able to simplify `DefinitionName` by no longer implementing everything needed by the interface `Type` - would be worth exploring.